pkg/cri/config/config_unix.go

```diff
@@ -104,5 +106,6 @@ func DefaultConfig() PluginConfig {
 		ImageDecryption: ImageDecryption{
 			KeyModel: KeyModelNode,
 		},
+		ImagePullProgressDeadline: time.Minute.String(),
```

Comment:
I am concerning if this may break current customer...

Reply:
yeah i am ok with this default. If there is no data transferred within 1 min, then most likely the image pull is gonna fail. 

That said, I am with Qiutong and would like to stay on the safer side. Can we make the default to 2m, which is the default `RuntimeRequestTimeout` used by kubelet ([code](https://github.com/kubernetes/kubernetes/blob/26da0ea91e1e08f7f8a9a600ff541c9f9314356d/pkg/kubelet/kubelet.go#L327)), so that this change will not change existing kubernetes behavior?

Reply:
@ruiwen-zhao If my memory is correct, the code you mentioned is used to connect to server, not for pulling request. And the kubelet doesn't add timeout for pulling request. That is why kubelet use `image_pull_progress_deadline(default=1m)` for dockershim.